.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "env": {
+        "commonjs": true,
+        "node": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 2015
+    },
+    "rules": {
+        "space-before-function-paren": ["error", {"anonymous": "always", "named": "never", "asyncArrow": "always"}],
+         "no-console": "off"
```

Comment:
the indent is different here, I suggest to use 2 spaces, without any oneliner, you can use `jq` or `JSON.stringify()` to get a proper formatted one ;)

Reply:
done.